internal/cli/cmd/install.go

```diff
@@ -55,7 +55,9 @@ cilium install --context kind-cluster1 --cluster-id 1 --cluster-name cluster1
 	cmd.Flags().MarkHidden("base-version")
 	cmd.Flags().StringVar(&params.DatapathMode, "datapath-mode", "", "Datapath mode to use")
 	cmd.Flags().StringVar(&params.IPAM, "ipam", "", "IP Address Management (IPAM) mode")
-	cmd.Flags().StringVar(&params.NativeRoutingCIDR, "native-routing-cidr", "", "CIDR within which native routing is possible")
+	cmd.Flags().StringVar(&params.IPv4NativeRoutingCIDR, "ipv4-native-routing-cidr", "", "IPv4 CIDR within which native routing is possible")
+	cmd.Flags().StringVar(&params.NativeRoutingCIDR, "native-routing-cidr", "", "IPv4 CIDR within which native routing is possible. Deprecated in favor of --ipv4-native-routing-cidr")
+	cmd.Flags().MarkDeprecated("native-routing-cidr", "use --ipv4-native-routing-cidr instead")
```

Comment:
Should we also hide this flag from usage messages, i.e. `cmd.Flags().MarkHidden("native-routing-cidr")`?

Reply:
[MarkDeprecated](https://github.com/cilium/cilium-cli/blob/2206e30ed9e81777413e047735602b1871fc1d5a/vendor/github.com/spf13/pflag/flag.go#L408-L422) will hide it.

Reply:
TIL, thanks!